docker/run.sh

```diff
@@ -18,13 +18,13 @@ docker run \
 # Volumes
 
 # Can also mount container file system (below commands can be repeated)
-# -v `~/Documents/back_end_plugins:/usr/src/application_manager_server/build/plugins -> ${local-path}{container-path}
+# -v `~/Documents/back_end_plugins:/usr/src/conforma_server/build/plugins -> ${local-path}{container-path}
 
 # Some common container paths
 
 # nginx config: /etc/nginx/conf.d/
 # nginx log: /var/log/nginx/
 # postgresql /etc/postgresql/12/main/
 # postgresql log: /var/log/postgresql/
-# back end snapshots: /usr/src/application_manager_server/database/snapshots/
-# back end and postgraphile log: /var/log/application_manager
+# back end snapshots: /usr/src/conforma_server/database/snapshots/
+# back end and postgraphile log: /var/log/conforma
```

Comment:
Would be good to test this with the instance for testing prior to updating our production server - even though we save the snapshot locally before re-uploading (and running a migration), I'm afraid it would fail due to changes on folders (and not getting the existing volumes

Reply:
I was planning on testing it with the 50004 instance as Jess wants the Fiji demo to go on there.